webapp/components/test-run.html

```diff
@@ -46,21 +46,27 @@
         width: 32px;
         height: 32px;
       }
+      .small img {
+        width: 24px;
+        height: 24px;
+      }
     </style>
 
-    <div>
-      <div><img src="{{displayLogo(testRun)}}" /></div>
-      <div>{{displayName(testRun.browser_name)}} {{shortVersion(testRun.browser_name, testRun.browser_version)}}</div>
-      <template is="dom-if" if="{{ !isDiff(testRun.browser_name) }}">
-        <div>{{displayName(testRun.os_name)}} {{testRun.os_version}}</div>
-        <div class="revision">@<a href="?sha={{testRun.revision}}">{{testRun.revision}}</a></div>
-        <div>{{dateFormat(testRun.created_at)}}</div>
-        <paper-tooltip offset=0>
-          {{displayName(testRun.browser_name)}} {{testRun.browser_version}}<br>
-          Labels: {{displayLabels(testRun.labels)}}<br>
-          {{moreTooltip(testRun)}}
-        </paper-tooltip>
+    <div class$='[[containerClass(small)]]'>
+      <div><img small="[[small]]" src="{{displayLogo(testRun)}}" /></div>
+      <template is="dom-if" if="[[!small]]">
```

Comment:
Though not easy to see, this wrapping `dom-if` is the only change here (and the `class$` on the container div above).